incubator/vault-operator/README.md

```diff
@@ -0,0 +1,141 @@
+# CoreOS vault-operator
+
+[vault-operator](https://coreos.com/blog/introducing-vault-operator-project) Simplify vault cluster
+configuration and management.
+
+__DISCLAIMER:__ While this chart has been well-tested, the vault-operator is still currently in beta.
+Current project status is available [here](https://github.com/coreos/vault-operator).
+
+## Introduction
+
+This chart bootstraps a vault-operator and allows the deployment of vault cluster(s). It depends on the `etcd-operator` being installed.
+
+## Official Documentation
+
+Official project documentation found [here](https://github.com/coreos/vault-operator)
+
+## Pre-requisites
+
+- Kubernetes 1.8+
```

Comment:
You may want to go for 1.9+ and use `apps/v1` for the deployment.

Reply:
We're stuck on 1.8, hence the `v1beta2`, but everybody else shouldn't suffer, so I'll bump.